docs/docs/metrics.md

```diff
@@ -143,11 +119,8 @@ expect the names of these metrics to change if the affected code changes.
 
 ### Derived metrics (mean, p99, ...)
 
-Our metrics library calculates derived metrics like "mean" and "p99". Unfortunately, if reported to statsd, they
-do not only relate to the reporting interval but the whole live time of the app with some exponential weighting
-algorithm. So try to build your dashboard around "counts" rather than "rates" where possible.
+Our metrics library calculates derived metrics like "mean" and "p99." Unfortunately, if they are reported to statsd, they do not only relate to the reporting interval but the whole live time of the app with some exponential weighting algorithm. So try to build your dashboard around "counts" rather than "rates" where possible.
```

Comment:
This refers again to the exponential decaying reservoirs:
https://dropwizard.github.io/metrics/3.1.0/manual/core/#exponentially-decaying-reservoirs